app/src/main/java/xyz/jhughes/laundry/apiclient/MachineConstants.java

```diff
@@ -0,0 +1,9 @@
+package xyz.jhughes.laundry.apiclient;
+
+/**
+ * Created by mvieck on 4/22/17.
+ */
+
+public class MachineConstants {
+    public static String API_ROOT = "http://laundry-api.sigapp.club";
```

Comment:
Can you add a comment here to explain why it isnt final? I feel like someone is going to get confused when they arent looking at how this is used in the tests.